list.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each
+    @list.length.time do |index|
```

Comment:
- [ ] I also got an error for an unknown method name `time`. I believe there is a typo here. The correct method is `times`.

See screenshot:
![a2](https://user-images.githubusercontent.com/39523312/155805144-0b788074-6563-436d-b1d8-58ce719ab46a.png)

Reply:
done

Reply:
- Hello @mmsesay, thanks for the insightful review. We made changes with reference to your actionable guidance. Hopefully, our project gets approved on this last attempt 🤞 
- Happy Coding!!